### PR TITLE
Add elements that make my page accessable and styling elements

### PR DESCRIPTION
**Pull request by asutherland91**
# PR Template

### Types of changes made?

- [x]  CSS
- [x]  HTML
- [x]  javascript
- [x]  functionality
- [x] testing

### What does this PR do?

- Adds functionality for accessibility compliance 
- Updates JS for adding a chart for amount spent per trip
- Writes test for new amount spent function
- Updates CSS and HTML to be more neat and well styled

### Questions/Comments/Relevant Screenshots?

I was attempting to also implement glide, but I was unable to figure it out at this time and decided to move along for the time being. I'm hoping to be able to come back to that later.

### CODE REVIEW

**Comment by nicktu12** [comment]
👍🏻

File: src/scripts.js [hunk 1]
```diff
@@ -145,6 +147,26 @@ function postNewTrip() {
     console.log(data)
     tripRepository.addNewTrip(data.newTrip)
     displayTripCards()
+  });
+};
+
+function displayAmountSpentChart(tripRepository, destinationRepository) {
```
Comment (hjawad22) [inline src/scripts.js:153]: Nice job incorporating a chart!

File: src/classes/Traveler.js [hunk 2]
```diff
@@ -20,6 +20,23 @@ class Traveler {
...
+    });
+      const amountSpentPerTrip = filteredTrips.map(trip => {
+        const destination = destinationRepository.getDestination(trip.destinationID);
+        let cost = 0;
+        cost += (trip.travelers * destination.estimatedFlightCostPerPerson);
+        cost += (trip.duration * destination.estimatedLodgingCostPerDay);
+        cost = (cost * 1.1).toFixed(2);
+        return {[destination.destination]: cost};
+       });
+      console.log(Object.values(amountSpentPerTrip[0]))
```
Comment (hjawad22) [inline src/classes/Traveler.js:37]: Make sure not to miss this console.log.

File: src/scripts.js [hunk 3]
```diff
@@ -145,6 +147,26 @@ function postNewTrip() {
...
+  const totalCost = traveler.calculateAmountSpentPerTrip(tripRepository, destinationRepository);
+  const labels = totalCost.map(trip => Object.keys(trip)[0]);
+  const data = totalCost.map(trip => Object.values(trip)[0]);
+  new Chart("chart", {
```
Comment (hjawad22) [inline src/scripts.js:157]: Nice job incorporating a chart!

**Comment by hjawad22** [comment]
Great work Adriane! I love the theme and implementing a chart is awesome. Keep up the great work! I think if you have time you can try to add glide but I think that you might not need it at this stage.

**Pull request merged**